src/index.js

```diff
@@ -649,28 +668,49 @@ async function processImageForScale(imageBuffer, scale, format, quality, maxScal
 /**
  * Create Contents.json for iOS assets
  */
-function createContentsJson(name, type = 'image') {
+function createContentsJson(name, format = 'png') {
```

Comment:
suggestion: Should we consider making the function name more related to iOS?

eg. `createAssetCatalogContentsJson`

Reply:
Sounds good, but I'd like to keep this PR smaller from unrelated changes. Let's find a chance in some future PR to update it.